README.md

```diff
@@ -347,6 +351,15 @@ User agents may only accept `Partitioned` cookies if their `SameSite` attribute
 User agents should reject any cookie set with both `Partitioned` and `SameParty` attributes.
 Since sites within the same First-Party Set are allowed access to unpartitioned `SameParty` cookies, the semantic is inconsistent with partitioned cookies.
 
+#### Limit the number of cookies a third party can use in a single partition
+
+A third-party domain's cookie jar should have a much lower per-partition size limit than existing garbage collection thresholds ([180 cookies per domain](https://source.chromium.org/chromium/chromium/src/+/master:net/cookies/cookie_monster.h;l=104;drc=da465ccade3a693e1deac3bf01b1c83d12dbf553) in Chrome).
+User agents must limit third-party domains to just one or some small number of cookies per-partition.
+Tthe number of cookies in a single partition per third-party is scoped by domain so that a third-party could not circumvent this limit by registering new subdomains.
```

Comment:
Typo: Tthe -> The

Reply:
Also, we need to consider implementation complexity involved in enforcing the limit per 3p domain, vs. per 3p origin.

Reply:
> Typo: Tthe -> The

Fixed.

> Also, we need to consider implementation complexity involved in enforcing the limit per 3p domain, vs. per 3p origin.

Do you mind elaborating?